tuts/create-your-first-substrate-chain/v2.0.0-alpha.6/interact.md

```diff
@@ -0,0 +1,101 @@
+2020-04-15 13:46:00 Substrate Node
+2020-04-15 13:46:00 ✌️  version 2.0.0-alpha.6-c1b33f8-x86_64-linux-gnu
+2020-04-15 13:46:00 ❤️  by Anonymous, 2017-2020
+2020-04-15 13:46:00 📋 Chain specification: Development
+2020-04-15 13:46:00 🏷  Node name: abrasive-sand-1388
+2020-04-15 13:46:00 👤 Role: AUTHORITY
+2020-04-15 13:46:00 ⛓  Native runtime: node-template-1:1(node-template-1)
+2020-04-15 13:46:00 🔨 Initializing Genesis block/state (state: 0x4603…c0c6, header-hash: 0x7fd5…54f7)
+2020-04-15 13:46:00 👴 Loading GRANDPA authority set from genesis on what appears to be first startup.
+2020-04-15 13:46:00 ⏱ Loaded block-time = 6000 milliseconds from genesis on first-launch
+2020-04-15 13:46:00 📦 Highest known block at #0
+2020-04-15 13:46:00 Using default protocol ID "sup" because none is configured in the chain specs
+2020-04-15 13:46:00 🏷  Local node identity is: QmStkPyqTUosZQvYyGzsQgbAnUv7xqE7VBYfThkePMzvuR
+2020-04-15 13:46:00 〽 Prometheus server started at 127.0.0.1:9615
+2020-04-15 13:46:05 💤 Idle (0 peers), best: #0 (0x7fd5…54f7), finalized #0 (0x7fd5…54f7), ⬇ 0 ⬆ 0
+2020-04-15 13:46:06 🙌 Starting consensus session on top of parent 0x7fd5647d42940dd960cd19612679eed979c542f42efb4b5078e8d00dbfd754f7
+2020-04-15 13:46:06 🎁 Prepared block for proposing at 1 [hash: 0x1ff8fd2f064782f65adabc3e79f75754b0cbb31ba5ea87e6ec37adcc2aa0d9fc; parent_hash: 0x7fd5…54f7; extrinsics (1): [0x851d…c946]]
+2020-04-15 13:46:06 🔖 Pre-sealed block for proposal at 1. Hash now 0x2fc528b959cad12872c478a397526ad119ff1d309fd3ddd33f7e15d1dee3d4da, previously 0x1ff8fd2f064782f65adabc3e79f75754b0cbb31ba5ea87e6ec37adcc2aa0d9fc.
+2020-04-15 13:46:06 🙌 Starting consensus session on top of parent 0x2fc528b959cad12872c478a397526ad119ff1d309fd3ddd33f7e15d1dee3d4da
+2020-04-15 13:46:06 ✨ Imported #1 (0x2fc5…d4da)
+2020-04-15 13:46:06 🎁 Prepared block for proposing at 2 [hash: 0x08ec27919011a0ce00ea94e1d3b9647b13392ce0643c8b10f292415b81c34b4f; parent_hash: 0x2fc5…d4da; extrinsics (1): [0xd56e…2407]]
+2020-04-15 13:46:06 🔖 Pre-sealed block for proposal at 2. Hash now 0x6e845b7ebeec35587cc0ff89c815d0930e30ec43e5271d4fa9ecedfbcd159c63, previously 0x08ec27919011a0ce00ea94e1d3b9647b13392ce0643c8b10f292415b81c34b4f.
+2020-04-15 13:46:06 ✨ Imported #2 (0x6e84…9c63)
+2020-04-15 13:46:10 💤 Idle (0 peers), best: #2 (0x6e84…9c63), finalized #0 (0x7fd5…54f7), ⬇ 0 ⬆ 0
+```
+
+If the number after `best:` is increasing, your blockchain is producing new blocks!
+
+## Start the Front End
+
+To interact with the local node we will use the Polkadot-js Apps user interface, often known as
+"Apps" for short. Despite the name, Apps will work with any Substrate-based blockchain including ours, not just Polkadot.
+
+In your web browser, navigate to [https://polkadot.js.org/apps](https://polkadot.js.org/apps/#/settings?rpc=ws://127.0.0.1:9944).
+
+On the `Settings` tab ensure that you are connected to a `Local Node` or `ws://127.0.0.1:9944`.
```

Comment:
How are we going to handle the change in the Weight type?
Transfers will not work on polkadot.js.org/apps unless we define `Weight as u32`

Reply:
This is solved with a change in Apps https://github.com/polkadot-js/apps/pull/2577

In the future we will give the node template a proper spec version.